src/Symfony/Bridge/PhpUnit/TextUI/Command.php

```diff
@@ -13,8 +13,10 @@
 
 if (version_compare(\PHPUnit\Runner\Version::id(), '6.0.0', '<')) {
     class_alias('Symfony\Bridge\PhpUnit\Legacy\CommandForV5', 'Symfony\Bridge\PhpUnit\TextUI\Command');
-} else {
+} elseif (version_compare(\PHPUnit\Runner\Version::id(), '6.0.0', '>=') && version_compare(\PHPUnit\Runner\Version::id(), '9.0.0', '<')) {
```

Comment:
the first part of the condition is useless. The `elseif` already ensures it is always true.